helm/templates/daemonset.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: {{ .Release.Name }}
```

Comment:
if we choose to use `{{ .Values.fullname }}`, all of name should be this.

Reply:
Update by https://github.com/merbridge/merbridge/pull/65#discussion_r810727669

I think maybe we just use values.fullname ?

Reply:
ok. so we should only use `.Values.fullname` rather than `.Release.Name` .